projects/humans/c3d/controllers/c3d_viewer/c3d_viewer.py

```diff
@@ -44,33 +45,195 @@
 }
 
 
-def isVirtualMarker(name):
-    """Return true is this is known to be a virtual marker."""
-    prefixList = [
-        'HED', 'LCL', 'LFE', 'LFO', 'LHN', 'LHU', 'LRA', 'LTI', 'LTO', 'PEL',
-        'RCL', 'RFE', 'RFO', 'RHN', 'RHU', 'RRA', 'RTI', 'RTO', 'TRX'
-    ]
-    virtualMarkers = ['CentreOfMass', 'CentreOfMassFloor']
-    for prefix in prefixList:
-        virtualMarkers.append(prefix + 'O')
-        virtualMarkers.append(prefix + 'A')
-        virtualMarkers.append(prefix + 'L')
-        virtualMarkers.append(prefix + 'P')
-    if name in virtualMarkers:
-        return True
-    return False
-
-
-def getPointsList(reader, name):
-    """Get a group of points and extract it's labels as a list of strings."""
-    if name not in reader.groups['POINT'].params:
-        return None
-    list = reader.groups['POINT'].get_string(name)
-    elementSize = reader.groups['POINT'].get(name).dimensions[0]
-    newlist = [list[i:i + elementSize] for i in range(0, len(list), elementSize)]
-    for i in range(len(newlist)):
-        newlist[i] = newlist[i].strip()
-    return newlist
+class c3dFile:
```

Comment:
It is a very good idea to create a class for this, it is now much cleaner, but the name of the class looks odd to me as at first glance it me looks like it is handling C3D files, but it is actually doing much more than that (C3D file reading is only a minor part of the class, it is also handling markers, robot-window communication, etc.)

Reply:
Yes, this class does a lot of things in relationship with a C3D file, so I believe that *c3dFile* is a good name for this class. I would like to avoid *c3dManager* or the like which sounds less precise to me. Do you have a better idea?

Reply:
Maybe simply `C3dViewer` as it is the name of the controller/file.
But that's not very important, I was just breafly confused when first reading the name of the class (maybe just a brief class description comment would be enough).

Reply:
> I would like to avoid c3dManager

I completely agree, in 99% of the cases, such kind of 'manager' keyword are completely useless.

Reply:
OK, I added some comment to better describe what this class does. `c3dFile` sounds better to me than `c3dViewer` as it basically loads and initializes a C3D file, but doesn't actually display it...